src/core/style/structure.scss

```diff
@@ -146,16 +146,19 @@ input {
   margin: 8px;
 }
 
-.md-virtual-scroll-container {
-  display: block;
-  position: relative;
-  overflow: hidden;
-  width: 100%;
-}
-
-.md-virtual-scroller {
-  width: 100%;
-  min-height: 100%;
+.md-scroll-mask {
+  position: absolute;
+  background-color: transparent;
+  > .md-scroll-mask-bar {
+    display: block;
+    position: absolute;
+    background-color: #fafafa;
```

Comment:
I don't think so because it's more of a browser emulation thing (most browsers don't allow styling the scroll bar color).